iphone/TitaniumKit/TitaniumKit/Sources/Modules/TiUIWindowProxy.m

```diff
@@ -404,10 +405,10 @@ - (void)setTitleAttributes:(id)args
   if (shouldUpdateNavBar && ([controller navigationController] != nil)) {
     UINavigationBar *navigationBar = controller.navigationController.navigationBar;
     if ([TiUtils boolValue:[self valueForKey:@"largeTitleEnabled"] def:NO]) {
-      if ([self shouldUseNavBarApperance]) {
-        navigationBar.standardAppearance.largeTitleTextAttributes = theAttributes;
-        navigationBar.scrollEdgeAppearance.largeTitleTextAttributes = theAttributes;
-      }
+      //      if ([self shouldUseNavBarApperance]) {
+      //        navigationBar.standardAppearance.largeTitleTextAttributes = theAttributes;
+      //        navigationBar.scrollEdgeAppearance.largeTitleTextAttributes = theAttributes;
+      //      }
```

Comment:
Is there a reason you deactivated it here? Does it have an influence of "normal" large titles (without additional title styling)?

Reply:
Yes, because originally titleAttributes applied style to both states(big and small), that condition is about - if largeTitleEnabled == true, apply style to both cases, else apply it to small one...